index.html

```diff
@@ -1,95 +1,115 @@
+    integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+</head>
+<title>FreeOTP</title>
+</head>
 
-      section {
-        padding: 0 4px 0 4px;
-        min-width: 200px;
-        flex: 1 1 0;
-      }
+<body data-bs-theme="dark">
+  <header class="bg-body-tertiary border-bottom container-fluid position-fixed">
+    <div class="container">
+      <nav class="d-flex navbar">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="">
+            <img class="rounded" id="logo" src="/img/freeotp.svg" alt="FreeOTP Logo">
+          </a>
+        </div>
+      </nav>
+    </div>
+  </header>
 
-      .screenshot img {
-        width: calc(100% - 2px);
-        border: 1px solid black;
-        display: block;
-      }
+  <main>
+    <section class="bg-body-secondary border-bottom d-flex align-items-center" id="welcome">
+      <div class="container p-5">
+        <div class="row">
+          <div class="col-6">
+            <h1>FreeOTP</h1>
+            <p class="text-secondary">
+              A free and open-source app for Android and iOS to generate one-time passwords.
+            </p>
+            <a href="/download.html" class="btn btn-primary">Get FreeOTP</a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="border-bottom d-flex align-items-center p-5" id="features">
+      <div class="container">
+        <div class="mb-5 text-center">
+          <h2>Why use FreeOTP?</h2>
+          <p class="text-secondary">Explore below to see why FreeOTP is a simple and secure mobile authenticator.</p>
+        </div>
+        <div class="row">
+          <div class="bg-body-tertiary col rounded m-2 p-4" id="security">
+            <div class="mb-4 text-center"><i class="bi bi-shield-fill fs-1"></i></div>
+            <h3>Security</h3>
+            <p class="text-secondary">FreeOTP uses <a href="https://www.ietf.org/rfc/rfc4226.txt">HOTP</a> and <a
+                href="https://www.ietf.org/rfc/rfc6238.txt">TOTP</a> by default to secure your tokens. This means that
+              no proprietary server-side component is necessary.</p>
+          </div>
+          <div class="bg-body-tertiary col rounded m-2 p-4" id="encryption">
+            <div class="mb-4 text-center"><i class="bi bi-qr-code-scan fs-1"></i></div>
+            <h3>Simple Setup</h3>
+            <p class="text-secondary">FreeOTP supports scaning a QR code to setup new tokens.</p>
+          </div>
+          <div class="bg-body-tertiary col rounded m-2 p-4" id="encryption">
+            <div class="mb-4 text-center"><i class="bi bi-fingerprint fs-1"></i></div>
+            <h3>Encryption</h3>
+            <p class="text-secondary">All of your tokens are encrypted by default. Just select a password that you would
```

Comment:
I would change this text to state that Tokens are stored in the device Keystore, enhancing security by making them more difficult to extract from the device.